src/math/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ if(CONFIG_SQRT_FIXED)
         add_local_sources(sof sqrt_int16.c)
 endif()
 
-if(CONFIG_EXP_FIXED)
-        add_local_sources(sof exp_fcn.c)
+if(CONFIG_MATH_EXP)
```

Comment:
If this is changed, should also the Kconfig name change?

Reply:
`Kconfig` is not using CONFIG_MATH_EXP but `EXP_FIXED` and default its set to `n`